src/parser.ml

```diff
@@ -145,40 +145,42 @@ module KW = Keywords.Make(
 let _ = KW.reserve "KIND"
 
 (** Keyword declarations. *)
-let _require_    = KW.create "require"
-let _open_       = KW.create "open"
-let _as_         = KW.create "as"
-let _let_        = KW.create "let"
-let _in_         = KW.create "in"
-let _symbol_     = KW.create "symbol"
-let _definition_ = KW.create "definition"
-let _theorem_    = KW.create "theorem"
-let _rule_       = KW.create "rule"
+let _TYPE_       = KW.create "TYPE"
```

Comment:
why so many changes in this file?

Reply:
I sorted the lines alphabetically

Reply:
Ok. Good.

Reply:
But TYPE should come later then... Please add at the beginning that it should be kept in alphabetical order.

Reply:
Capital letters come first I guess (I only used M-x sort-lines on emacs...)

Reply:
I guess that Emacs is using ASCII code ordering. This is not very human friendly. I propose to use the standard lexicographic ordering ignoring whether letters are small or big.